tigerpath/majors_and_certificates/majors/SPO_2018.json

```
@@ -0,0 +1,359 @@
+{
+  "type": "Major",
+  "name": "Spanish and Portuguese",
+  "code": "SPO",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-spanish-and-portuguese-languages",
+    "https://spo.princeton.edu/undergraduate/concentration-new"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Pedro Meira Monteiro",
+      "email": "pmeira@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Christina H. Lee",
+      "email": "chrislee@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Concentration",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": null,
+      "req_list": [
+        {
+          "name": "Spanish",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Normally, students complete two 200-level courses in Spanish or one 200-level course in Portuguese.",
+              "course_list": [

```

I don't think the prerequisites are excluded from also being departmentals are they? As long as they're 210 and above, right? Then again, I may be wrong.
```suggestion
              "double_counting_allowed": true,
              "course_list": [
```